### PR TITLE
Convert README to rst

### PR DESCRIPTION
**Pull request by adamchainz**
So it doesn't look bad on PyPI like it currently does: https://pypi.python.org/pypi/ua-parser

After merging, `python setup.py register` should be run to upload the new version without waiting for release.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,9 +1,15 @@
 [tox]
-envlist = py26, py27, pypy, py31, py32, py33, py34, py35, docs, pep8, py3pep8
```
Comment (mattrobenolt) [inline tox.ini]: Why did you remove these? This is breaking CI.

Comment (adamchainz) [inline tox.ini]: because they don't have any `testenv` defined in `tox.ini` so they just rerun the tests

Comment (adamchainz) [inline tox.ini]: removed from `.travis.yml`

**Comment by mattrobenolt** [comment]
Ah, I see. I'll create a new ticket to bring back pep8. Thanks for pointing that out.

**Comment by adamchainz** [comment]
You mean `pycodestyle` now 😛

**Comment by mattrobenolt** [comment]
> You mean `pycodestyle` now 😛

ugh, I will likely just use flake8 since that's what I know.

**Comment by adamchainz** [comment]
It still depends on `pycodestyle` :)

**Pull request merged**